src/common/hook/usePublishQueue/usePublishQueue.tsx

```diff
@@ -41,15 +43,25 @@ export const usePublishQueue = (
     };
   });
 
+  const pendingPublishDocuments = pendingJobIndex.reduce(
+    (acc, curr) => [...acc, ...jobs[curr].documents],
+    [] as WrappedDocument[]
+  );
+
```

Comment:
Suggest to use .map as it's easier to read (like `failedPublishedDocuments`), can change the `publishedDocuments` also

Reply:
ok